mentat/config.py

```diff
@@ -146,13 +147,14 @@ class Config:
         },
     )
 
-    theme: str | None = attr.field(
+    theme: str | None = attr.field(  # pyright: ignore
         default="light",
         metadata={
             "description": (
                 "Theme for interaction possible choices are light, dark or none."
             )
         },
+        validator=validators.in_(Themes.keys.append(None)),  # pyright: ignore
```

Comment:
Why is there an append None? And I don't think append returns a value, so I don't think this works.

Reply:
fixed.